src/app/services/theme.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { Theme } from '../enum/theme';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class ThemeService {
+  private activeTheme: Theme = Theme.light;
```

Comment:
if you want always by default load dark theme - change here:
` private activeTheme: Theme = Theme.light;`
to ` private activeTheme: Theme = Theme.dark;`